packages/schemer/src/index.ts

```diff
@@ -243,17 +239,9 @@ export default class Schemer {
     }
   }
 
-  async _validateAssetAsync({
-    fieldPath,
-    data,
-    meta,
-  }: {
-    fieldPath: string,
-    data: string,
-    meta: Meta,
-  }) {
+  async _validateAssetAsync({ fieldPath, data, meta }: Asset) {
     if (meta && meta.asset && data) {
-      if (meta.asset.contentTypePattern && meta.asset.contentTypePattern.startsWith('^image')) {
+      if (meta.contentTypePattern && meta.contentTypePattern.startsWith('^image')) {
```

Comment:
Yeah, I fixed this because the original code seemed to be wrong in an obvious way that was caught by the TypeScript types (yay!). (The underlying data structure looks like this: https://github.com/expo/universe/blob/68816802d45e7941ea781c80a3004a8dd7e27f7a/server/www/xdl-schemas/UNVERSIONED-schema.json#L72-L74).

The code was introduced by a recent commit https://github.com/expo/expo-cli/commit/7f1860da9cdda9128b715724182c91d30e30b698.

However, I didn't test my fix at runtime yet. I can test it and extract the change to a separate PR.